itest/rust/src/register_tests/constant_test.rs

```diff
@@ -174,6 +174,8 @@ godot::sys::plugin_add!(
             register_methods_constants_fn: ::godot::private::ErasedRegisterFn {
                 raw: ::godot::private::callbacks::register_user_methods_constants::<HasOtherConstants>,
             },
+            #[cfg(all(since_api = "4.3", feature = "docs"))]
+            docs: None
```

Comment:
this isnt going to compile, itest doesnt have a docs feature

Reply:
well it doesnt change anything, because it can never be enabled

Reply:
Makes sense, missed leftover – I removed it since itest probably won't be called with `godot-core/docs`/`godot/register-docs` anyway